docs/src/1-Installation.doc.js

```diff
@@ -0,0 +1,259 @@
+// @flow strict
```

Comment:
@AlbertCarreras let's avoid the numbering of the files.

Reply:
done